src/html/index.html

```diff
@@ -4,9 +4,25 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello World!</title>
+    <title>Chat app</title>
   </head>
   <body>
-    <h1>Hello world!</h1>
+    <header>Chat</header>
```

Comment:
A `header` element defines that the content is heading information for the landmark area, in this case the `body`. It does not semantically define the content itself. This still should have an `<h1>` element inside the `<header>` to provide that definition.

Reply:
Aforementioned semantic HTML change made.